a3p-integration/proposals/c:stake-bld/stakeBld.test.js

```diff
@@ -66,10 +66,31 @@ test('basic', async t => {
     },
   });
 
-  const postDelegation = await currentDelegation();
-  t.is(postDelegation.length, 2, 'new delegation now');
-  t.like(postDelegation[1], {
+  const afterDelegate = await currentDelegation();
+  t.is(afterDelegate.length, 2, 'delegations after delegation');
+  t.like(afterDelegate[1], {
     balance: { amount: '10', denom: 'ubld' },
     // omit 'delegation' because it has 'delegatorAddress' which is different every test run
   });
+
+  await walletUtils.broadcastBridgeAction(GOV1ADDR, {
+    method: 'executeOffer',
+    offer: {
+      id: 'request-undelegate',
+      invitationSpec: {
+        source: 'continuing',
+        previousOffer: 'request-stake',
+        invitationMakerName: 'Undelegate',
+        invitationArgs: [VALIDATOR_ADDRESS, { brand: brand.BLD, value: 5n }],
+      },
+      proposal: {},
+    },
+  });
+
+  // TODO find a way to wait until completionTime, so we can check the count has gone back down
```

Comment:
We can query and then wait like we do for blocks etc.

It's 21 days, so we probably want to reduce it to something like 15sec first.

```
$ agd query staking params -o json | jq  .unbonding_time
"1814400s"
```